src/tests/Currency.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
As mentioned above and for the same reasons, kindly remove in-line linters disable.

There are a couple of other occurrences, please modify them too.